src/Users/Users/examples/v1.0/Get-MgUserOauth2PermissionGrant.md

```diff
@@ -1,18 +1,10 @@
-### Example 1: {{ Add title here }}
+### Example 1: Get user's oAuth2PermissionGrant 
 ```powershell
-PS C:\> {{ Add code here }}
+PS C:\> Get-MgUserOauth2permissionGrant  -UserId "68550c41-18a3-495e-aa1d-cf628c171b48"
 
-{{ Add output here }}
+Id                                                               ClientId                             ConsentType PrincipalId                          ResourceId                           Scope
+--                                                               --------                             ----------- -----------                          ----------                           -----
+F-eYWQGKuUS7WYM4IyWyuchvEPl3fYZLhdmaX2Ic7rRBDFVooxheSaodz2KMFxtI 5998e717-8a01-44b9-bb59-83382325b2b9 Principal   68550c41-18a3-495e-aa1d-cf628c171b48 f9106fc8-7d77-4b86-85d9-9a5f621ceeb4  User.Read offline_access profile
```

Comment:
I suggest piping the output to `Format-List` so that the output fits on the page better.